.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
This is platform specific, and should probably be in _your_ `.gitignore`, not every repo's. I'm happy to let it go, though.